### PR TITLE
rm -fr lib/pf/firewallsso/

### PR DESCRIPTION
**Pull request by julsemaan**
# Description
Delete lib/pf/firewallsso/*

# Impacts
Old firewall SSO code

# Delete branch after merge
YES

## NOTE: Although I took great care in finding the old references to the modules, I may have missed some. It would be good if someone could check if I didn't forget a reference somewhere

### CODE REVIEW

**Comment by jrouzierinverse** [comment]
Did you run the smoke test?

**Comment by julsemaan** [comment]
smokes are b0rken on my dev machine so I wasn't able to test them on this code

I'm working on getting that to work again

File: addons/dev-helpers/dump.pl [hunk 1]
```diff
@@ -151,16 +151,6 @@ sub _run {
...
-package pf::dump::firewallsso;
-use base qw(pf::dump::cmd);
-use Data::Dumper;
-__PACKAGE__->mark_as_loaded();
-
-sub _run {
-    require pf::config;
-    print Data::Dumper::Dumper(\%pf::config::ConfigFirewallSSO);
-}
-
```
Comment (jrouzierinverse) [inline addons/dev-helpers/dump.pl]: Why was this removed we still have a firewall sso config.

**Comment by jrouzierinverse** [comment]
smoke test passed.
Except for what I mentioned everything looks good.

**Comment by julsemaan** [comment]
I've reverted the dump.pl removal
